src/main/java/org/apache/sling/feature/launcher/impl/Main.java

```diff
@@ -174,8 +179,9 @@ protected static void parseArgs(final LauncherConfig config, final String[] args
         final Option fwkProperties = Option.builder(OPT_FRAMEWORK_PROPERTIES)
                 .longOpt("framework-properties")
```

Comment:
Perhaps this name (and description) should be changed to "framework-property" since each instance of the arg supplies just one <name>=<value> pair?

Reply:
If it is really just one property than it in right. Will fix tomorrow

Reply:
But that means that we could only delegate one framework property set as an env param to the cli.

This might be a Problem later.

Reply:
Ok, I now see how the Dockerfile was using those ENV configurations and it looks like it may be assuming that -D accepts a list of name=value properties?

I could be wrong, but I'm not seeing where Apache cli automatically handles a list of properties in that way.  And as far as I know, defining a system property with a "-D" jvm arg expects only a single name=value and not a list.

Reply:
I was absolutely sure! But absolutly Wrong.

I remembered false that here was described that pattern. with the csv.
https://commons.apache.org/proper/commons-cli/javadocs/api-1.3.1/org/apache/commons/cli/CommandLine.html#getOptionProperties-java.lang.String-

Will change all to the single -Dkey=value

Thank you

Reply:
looks good to me.  

Also, it would be good to say what that the syntax of the arg is "name=value" in the printHelp output for the option